### DISCUSSION TITLE
Why is there no <Video />

### DISCUSSION THREAD

**Discussion by Vukichinuky** [vercel/next.js#47120] [Help]
### Summary

Is there any reason there is no Video in nextjs like there is Image, Link..
This would be great. 
I cant find anything in your documentation where Video is mentioned. What is best way to have video in background ?

**Comment by andrepichardo** [comment]
Hi @Vukichinuky!

Although Next.js doesnt have a ```<Video/>``` component, you can use the basic HTML ```<video/>``` tag like this (example using TailwindCSS):

```
<video 
preload="auto"
playsInline 
autoPlay
muted 
loop
className='w-full h-screen'
>
 <source src="/video.mp4" type="video/mp4" /> 
</video>
```

Hope it helps! :)
